src/api/spec/models/project_spec.rb

```diff
@@ -97,6 +97,54 @@
     it { expect(tumbleweed_project.image_template?).to be(false) }
   end
 
+  describe '#branch_remote_repositories' do
+    let (:package) { create(:package) }
+    let (:remote_project) { create(:remote_project) }
+
+    it 'returns false if the interconnect could not be found' do
+      expect(project.branch_remote_repositories("invalid_remote_project", package)).to eq(false)
+    end
+
+    it 'updates a project meta description' do
```

Comment:
I would like to see some more tests:

1) A RemoteProject _meta file without repositories (so that it does not add any repository)
2) A LocalProject _meta file with already existing repositories (so it does append the remote repositories)